modules/rostests/apitests/ntdll/NtQueryInformationFile.c

```diff
@@ -20,4 +20,36 @@ START_TEST(NtQueryInformationFile)
     Status = NtQueryInformationFile(NULL, NULL, NULL, 0, 0x80000000);
     ok(Status == STATUS_INVALID_INFO_CLASS ||
        ntv6(Status == STATUS_NOT_IMPLEMENTED), "Status = %lx\n", Status);
+
+    /* Get the full path of the current executable */
+    CHAR Path[MAX_PATH];
+    DWORD Length = GetModuleFileNameA(NULL, Path, MAX_PATH);
+    ok(Length != 0, "GetModuleFileNameA failed\n");
+    if (Length == 0)
+        return;
+
+    /* Open the file */
+    HANDLE hFile = CreateFileA(Path,
+                               GENERIC_READ,
+                               FILE_SHARE_READ,
+                               NULL,
+                               OPEN_EXISTING,
+                               FILE_ATTRIBUTE_NORMAL,
+                               NULL);
+    ok(hFile != INVALID_HANDLE_VALUE, "CreateFileA failed\n");
+    if (hFile == INVALID_HANDLE_VALUE)
+        return;
+
+    /* Query FileEndOfFileInformation */
+    FILE_END_OF_FILE_INFORMATION EndOfFileInformation;
```

Comment:
I'm not a big fan of variables declarations in the middle of code in C.

Reply:
And I am not a big fan of style nit-picking in PRs.